tests/test_unittest/test_gtscript_frontend.py

```diff
@@ -755,7 +755,7 @@ def definition_func(
         )
 
 
-class TestFunctionReturn:
+class TestFunctions:
```

Comment:
Not sure about this, there's already a `TestFunctions` in the same file. I think this class collects tests that check the appropriate placement of `return` statements, so the tests you added should be moved elsewhere.